src/main/java/org/mskcc/kickoff/converter/SampleSetToRequestConverter.java

```diff
@@ -159,10 +173,14 @@ private void setSamples(KickoffRequest kickoffRequest, KickoffSampleSet sampleSe
         DEV_LOGGER.info(String.format("Samples found for sample set: %s [%s]", sampleSet.getName(), Utils
                 .getJoinedCollection(sampleSetSamples.keySet())));
 
-        for (ExternalSample externalSample : sampleSet.getExternalSamples()) {
-            sampleSetSamples.put(externalSample.getIgoId(), externalSample);
-        }
+//        for (KickoffExternalSample externalSample : sampleSet.getExternalSamples()) {
```

Comment:
not deleting makes less beautiful (I know my code is ugly)

Reply:
I don't know how it ended there, I wanted to delete it